es-ds-components/package.json

```diff
@@ -11,5 +11,9 @@
   "license": "MIT",
   "devDependencies": {
     "nuxt": "^3.12.4"
+  },
+  "dependencies": {
+    "@nuxtjs/google-fonts": "^3.2.0",
+    "nuxt-primevue": "^3.0.0"
```

Comment:
> Not totally sure how other components were working without installing the prime-vue dependency in es-ds-components

Because es-ds-components is not actually responsible for building things.. instead the downstream repo such as es-ds-docs does it

Reply:
No harm in including it though